xlive/Blam/Engine/Simulation/MachineID.h

```diff
@@ -2,5 +2,13 @@
 #pragma once
 struct s_machine_identifier
 {
-	char gap[6];
```

Comment:
this is probably the abEnet, just do `byte machine_identifier[6]`

Reply:
Yea its actually abNet only but bungie used `s_machine_identfier` as their goto structure in a lot of places instead declaring a 6 byte long array everywhere so we should keep this structure

Reply:
meant it should actually be defined as `byte machine_identifier[6]` in that structure

Reply:
done